typescript-selenium-webdriver-sample/tests/index.test.ts

```diff
@@ -116,25 +118,54 @@ describe('index.html', () => {
         targets: violation.nodes.map(node => node.target),
     });
 
+    // Export both SARIF and HTML results files
+    async function exportResults(baseFileName: string, axeResults: Axe.AxeResults): Promise<void> {
+        // This directory should be .gitignore'd and should be published as a build artifact in azure-pipelines.yml
+        const testResultsDirectory = path.join(__dirname, '..', 'test-results');
+        await promisify(fs.mkdir)(testResultsDirectory, { recursive: true });
+
+        const baseResultsFile = path.join(testResultsDirectory, baseFileName);
+        await exportAxeAsSarifTestResult(baseResultsFile + '.sarif', axeResults);        
+        await exportAxeAsHtmlReport(baseResultsFile + '.html', axeResults);        
+    }
+
+    // accessibility-insights-report provides a friendly, human-readable, and accessibile report in a downloadable .html file.
+    async function exportAxeAsHtmlReport(htmlReportFile: string, axeResults: Axe.AxeResults): Promise<void> {
+
+        const pageTitle = await driver.getTitle();
+        const browserSpec = await driver.executeScript("return navigator.userAgent") as string;
```

Comment:
The axe results v1 format (the default produced by axe) already embeds the user agent in the axe results as `$.testEnvironment.userAgent`; can the reporter infer from that rather than requiring an extra script execution?

Reply:
I didn't know it was in the results. We can remove the need to have it in the report request then.